doc/user/layouts/shortcodes/session-variables.html

```diff
@@ -20,6 +20,7 @@
 emit_timestamp_notice                       | `false`                   | Boolean flag indicating whether to send a `notice` specifying query timestamps.
 emit_trace_id_notice                        | `false`                   | Boolean flag indicating whether to send a `notice` specifying the trace ID, when available.
 enable_session_rbac_checks                  | `false`                   | **Read-only.** Boolean flag indicating whether RBAC is enabled for the current session.
+emit_introspection_query_notice             | `true`                    | Whether to print a notice when querying replica introspection relations.
```

Comment:
Can you also make the description consistent with the other `emit_*` variables and move it into its alphabetical position?